src/libraries/System.Private.CoreLib/src/System/Environment.Windows.cs

```diff
@@ -2,10 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Collections.Generic;
+using System.ComponentModel;
```

Comment:
Is this using still needed?

Reply:
I don't think this is needed anymore. I removed it.